packages/runner-ct/src/lib/state.ts

```diff
@@ -194,7 +196,20 @@ export default class State {
   }
 
   @action setSpec (spec: Cypress.Cypress['spec'] | null) {
-    this.spec = spec
+    if (this.isInitialBuildSucceed) {
```

Comment:
nit: I think `hasInitialBuildSucceeded` is a little more natural.

Reply:
I love the convention of all the boolean variables starting with `is` like previous `isRunning`, `isLoading` – it is much easier to understand the type without thinking :) 

Maybe you'll find better name starting with `is` for this variable? Thank you

Reply:
How about `isInitialBuildSuccesful` ? Not perfect temporality because it "was" succesful but I prefer it.